.appveyor.yml

```diff
@@ -0,0 +1,29 @@
+clone_depth: 2
+clone_folder: c:\projects\sentry-php
+
+cache:
+  - composer.phar
+
+init:
+  - SET PATH=c:\php;%PATH%
+  - SET COMPOSER_NO_INTERACTION=1
+  - SET SYMFONY_DEPRECATIONS_HELPER=strict
+  - SET ANSICON=121x90 (121x90)
+
+install:
+  - mkdir c:\php && cd c:\php
+  - appveyor DownloadFile https://github.com/symfony/binary-utils/releases/download/v0.1/php-7.1.3-Win32-VC14-x86.zip
```

Comment:
That would be really slow. I wouldn't burden AppVeyor with a full matrix, we're already testing every version on Travis (which is faster due to the fact that PHP is preinstalled).

Not even Symfony builds for multiple versions under windows! It seems an overkill to me.

Reply:
>That would be really slow

Since Travis CI would run in parallel with AppVeyor it would not be much slower imho. The point in testing all versions is that you may have a bug in a specific version of PHP for Windows which may not exists in Linux

Reply:
Matrix is a bit hard for now to put up, we need to handle the PHP installation manually in each instance.

Skipping for now, we can cover that in a separate PR if we want.